gw_spaceheat/actors/i2c_relay_multiplexer.py

```diff
@@ -100,56 +108,82 @@ def initialize_board(self) -> None:
-                        self.krida_relay_pin[gw_idx] = SimulatedPin(
-                            value=KridaPinState.DeEnergized.value
+                        self.logger.info(f"Found board at {address} for board {board_idx}")
+                    except Exception as e:
+                        self.logger.warning(
+                            f"Failed to get board at {address} for board {board_idx}: {e}"
                         )
-                    self._send_to(self.atn,
-                                  Glitch(
-                                      FromGNodeAlias=self.layout.scada_g_node_alias,
-                                      Node=self.node.Name,
-                                      Type=LogLevel.Warning,
-                                      Summary=f"i2c board {board_idx} ({hex(address)}) failed to initialize. Setting as simulated",
-                                      Details=f"I2c Errors: {e}",
-                                  )
-                    )
+                        await asyncio.sleep(wait_s)
+                        continue
+
+                    await asyncio.sleep(0.2)
+                    self.logger.info(f"initializing board {board_idx} at {hex(address)}")
+                    try:
+                        for j in range(1, 17):
+                            # set relay to correct pin, and switch to output. This energizes all relays
+                            gw_idx = (board_idx - 1) * 16 + j
+                            pin_idx = gw_to_pin(gw_idx)
+                            self.krida_relay_pin[gw_idx] = self.krida_board[board_idx].get_pin(
+                                pin_idx
+                            )
+                            self.krida_relay_pin[gw_idx].switch_to_output()
+                        for j in range(1, 17):
+                            # move all relays back to de-energized position
+                            self.krida_relay_pin[
+                                i * 16 + j
+                            ].value = ChangeKridaPin.DeEnergize.value
+                        # and record the de-energized state for all known relays
+                        self.logger.info(f"Successfully initialized board {board_idx}")
+                        setup_done = True
+                    except Exception as e:
+                        self.logger.warning(f"Trouble initializing board {board_idx}! {e}")
+                        setup_attempts += 1
+                        await asyncio.sleep(wait_s)
+                # send up a Glitch if things took more than once
+                if setup_attempts > 0:
+                    if not setup_done:
+                        for j in range(1,17):
+                            gw_idx = (board_idx - 1) * 16 + j
+                            self.krida_relay_pin[gw_idx] = SimulatedPin(
+                                value=KridaPinState.DeEnergized.value
+                            )
+                        log_level = LogLevel.Critical
```

Comment:
This Glitch should turn into an alert - it means we are simulating one of the relay boards. Moderately better than the scada rebooting constantly but still an immediate problem